index.html

```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work2">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work3">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+    
+    <section class="works-template work4">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work5">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
     
+    <section class="works-template work6">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
+          <div class="tags tag1">Ruby on Trains</div>
+          <div class="tags tag2">Awesome minds</div>
+          <div class="tags tag3">JavaScript</div>
+          <div class="tags tag4">Rubic</div>
+        </div>
+        <button class="btn-more-project">See Project</button> 
+      </div>
+    </section>
+
+  </div>
+
+<!--About me section-->
+  <section class="aboutme-container">
+    <h1 class="aboutme-h1">About me</h1>
```

Comment:
- [ ] Replace the `h1` with an `h2`. The h1 tag is just for the title of the website.